src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol_ImplementationChecks.cs

```diff
@@ -131,7 +131,7 @@ private void CheckAbstractClassImplementations(BindingDiagnosticBag diagnostics)
 
                 HasBaseTypeDeclaringInterfaceResult? hasBaseClassDeclaringInterface = null;
 
-                foreach (var interfaceMember in @interface.GetMembersUnordered())
+                foreach (var interfaceMember in @interface.GetMembers())
```

Comment:
Could you please explain the impact of this change?

Reply:
> Could you please explain the impact of this change?

We collect and emit MethodImpl entries in deterministic order.